Source/Diagnostics/Diagnostics.H

```diff
@@ -131,8 +131,10 @@ protected:
     int nmax_lev; /**< max_level to allocate output multifab and vector of field functors. */
     /** Number of levels to be output*/
     int nlev_output;
-    /** Name of species to write to file */
-    std::vector< std::string > m_species_names;
+    /** Names of species to write to output */
+    std::vector< std::string > m_output_species_names;
+    /** Names of all species in the simulation */
+    std::vector< std::string > m_all_species_names;
     /** Each element of this vector handles output for 1 species */
     amrex::Vector< ParticleDiag > m_all_species;
```

Comment:
I think that the size of these vectors are:
 - `m_output_species_names` : number of output species
 - `m_all_species_names` : total number of species in the simulation
 - `m_all_species` : number of output species

Is that correct? If so, using the word `all` for the last vector is a bit misleading no? I would rename it `m_output_species` to keep the names consistent.